pandas/io/common.py

```diff
@@ -169,6 +169,14 @@ def is_s3_url(url):
         return False
 
 
+def is_gcs_url(url):
+    """Check for a gcs url"""
+    try:
+        return parse_url(url).scheme in ['gcs', 'gs']
+    except:  # noqa
```

Comment:
Make this `except Exception`, or even better figure out what `parse_url` can raise, and catch those (plus `AttributeError`).

Reply:
From looking over `urlparse` it seems that it will also only raise `AttributeError` so I used that. For consistency I changed the other url parsers `except`s as well.